From a2fadf980add7cddb7195379931387aa8112a442 Mon Sep 17 00:00:00 2001
From: Denis 'GNUtoo' Carikli <GNUtoo@no-log.org>
Date: Fri, 8 Jun 2012 23:11:09 +0200
Subject: [PATCH] fsobasics: fix threading compilation with vala 0.16.0

Without that fix we have:
  threading.vala:77.53-77.55: error: Cannot convert from `int' to `void*'
                                         void* param = 0x0 )

Signed-off-by: Denis 'GNUtoo' Carikli <GNUtoo@no-log.org>
---
 fsobasics/threading.vala |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fsobasics/threading.vala b/fsobasics/threading.vala
index a9e697c..dd2caaf 100644
--- a/fsobasics/threading.vala
+++ b/fsobasics/threading.vala
@@ -74,7 +74,7 @@ public bool isMainThread()
 
 public void callDelegateOnMainThread( VoidFuncWithVoidStarParam func,
                                       bool waitForCompletion = false,
-                                      void* param = 0x0 )
+                                      void* param = null )
 {
     var mutex = new GLib.Mutex();
     var cond = new GLib.Cond();
-- 
1.7.8.6

